lib/rules/no-unmodified-loop-condition.js

```diff
@@ -261,10 +261,10 @@ module.exports = function(context) {
 
             for (var j = 0; j < condition.groups.length; ++j) {
                 var be = condition.groups[j];
-                var group = groupMap.get(be);
+                var group = groupMap[be];
```

Comment:
Then it is a test problem to begin with. Can you update the tests to make this fail?

> On Jan 21, 2016, at 21:12, Toru Nagashima <notifications@github.com> wrote:
> 
> In lib/rules/no-unmodified-loop-condition.js:
> 
> > @@ -261,10 +261,10 @@ module.exports = function(context) {
> >  
> >              for (var j = 0; j < condition.groups.length; ++j) {
> >                  var be = condition.groups[j];
> > -                var group = groupMap.get(be);
> > +                var group = groupMap[be];
> be is not strings.
> 
> —
> Reply to this email directly or view it on GitHub.
>